app/scripts/background.js

```diff
@@ -331,6 +332,34 @@ function maybeDetectPhishing(theController) {
   );
 }
 
+/**
+ * Overrides the Content-Security-Policy Header, acting as a workaround for an MV2 Firefox Bug.
```

Comment:
Can you add a link to the firefox CSP issue, https://bugzilla.mozilla.org/show_bug.cgi?id=1446231, here?